.gitignore

```diff
@@ -99,6 +99,6 @@ src/stamp-h1
 /bazel-*
 
 zig-out/
-zig-cache/
```

Comment:
can we keep this so it also works for zig 0.12.1?

Reply:
Yes of course :+1: